[2025-03-21T20:27:14.760+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: external_customer_pipeline.store_customer_data_to_hdfs_and_hive manual__2025-03-21T20:24:34.002999+00:00 [queued]>
[2025-03-21T20:27:14.775+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: external_customer_pipeline.store_customer_data_to_hdfs_and_hive manual__2025-03-21T20:24:34.002999+00:00 [queued]>
[2025-03-21T20:27:14.776+0000] {taskinstance.py:1359} INFO - Starting attempt 2 of 3
[2025-03-21T20:27:14.795+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): store_customer_data_to_hdfs_and_hive> on 2025-03-21 20:24:34.002999+00:00
[2025-03-21T20:27:14.805+0000] {standard_task_runner.py:57} INFO - Started process 2158 to run task
[2025-03-21T20:27:14.812+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'external_customer_pipeline', 'store_customer_data_to_hdfs_and_hive', 'manual__2025-03-21T20:24:34.002999+00:00', '--job-id', '906', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_batch.py', '--cfg-path', '/tmp/tmpg2zu92zs']
[2025-03-21T20:27:14.815+0000] {standard_task_runner.py:85} INFO - Job 906: Subtask store_customer_data_to_hdfs_and_hive
[2025-03-21T20:27:14.877+0000] {task_command.py:415} INFO - Running <TaskInstance: external_customer_pipeline.store_customer_data_to_hdfs_and_hive manual__2025-03-21T20:24:34.002999+00:00 [running]> on host 4f421949e8a6
[2025-03-21T20:27:15.076+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='external_customer_pipeline' AIRFLOW_CTX_TASK_ID='store_customer_data_to_hdfs_and_hive' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T20:24:34.002999+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T20:24:34.002999+00:00'
[2025-03-21T20:27:18.272+0000] {pipeline_batch.py:217} ERROR - HDFS put command failed: put: `/data/customer/customer_20250321_202434.json': File exists

[2025-03-21T20:27:18.273+0000] {pipeline_batch.py:222} ERROR - Error writing to HDFS: HDFS put command failed: put: `/data/customer/customer_20250321_202434.json': File exists

[2025-03-21T20:27:18.700+0000] {pipeline_batch.py:44} INFO - Discord message sent successfully
[2025-03-21T20:27:18.703+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/airflow/dags/pipeline_batch.py", line 218, in store_customer_data_to_hdfs_and_hive
    raise RuntimeError(error_message)
RuntimeError: HDFS put command failed: put: `/data/customer/customer_20250321_202434.json': File exists


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/pipeline_batch.py", line 223, in store_customer_data_to_hdfs_and_hive
    raise RuntimeError(error_message)
RuntimeError: Error writing to HDFS: HDFS put command failed: put: `/data/customer/customer_20250321_202434.json': File exists

[2025-03-21T20:27:18.723+0000] {taskinstance.py:1398} INFO - Marking task as UP_FOR_RETRY. dag_id=external_customer_pipeline, task_id=store_customer_data_to_hdfs_and_hive, execution_date=20250321T202434, start_date=20250321T202714, end_date=20250321T202718
[2025-03-21T20:27:18.756+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 906 for task store_customer_data_to_hdfs_and_hive (Error writing to HDFS: HDFS put command failed: put: `/data/customer/customer_20250321_202434.json': File exists
; 2158)
[2025-03-21T20:27:18.783+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-03-21T20:27:18.855+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
